application-services/custom/camera-management/README.md

```diff
@@ -256,6 +269,16 @@ The API log shows the status of the 5 most recent calls and commands that the ma
 
 ![inference events](./images/inference-events.png)  
 
+### Inference results in Edgex
+
+To view inference results in Edgex, open Edgex UI [http://localhost:4000](http://localhost:4000) and view data streaming
+under `Event Data Stream` located under `DataCenter` tab.
```

Comment:
Maybe add a direct link
http://localhost:4000/en-US/#/core-data/event

and tell them to click the start button in the `Event Data Stream`.

Reply:
for some reason http://localhost:4000/en-US/#/core-data/event takes to the dashboard only.

Reply:
Yeah looks that way. Weird.